RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -344,15 +346,7 @@ void PulseShapeFitOOTPileupCorrection::apply(const CaloSamples & cs,
    std::vector<float> fitParsVec;
 
    if(tstrig >= ts4Min_&& tsTOTen > 0.) { //Two sigma from 0
-     pulseShapeFit(energyArr, pedenArr, chargeArr, pedArr, gainArr, tsTOTen, fitParsVec, noiseADCArr);
-   }
-   else if((tstrig < ts4Min_||tsTOTen < 0.)&&(ts4Min_==0)){
-     fitParsVec.clear();
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(0.);
-     fitParsVec.push_back(999.);
-     fitParsVec.push_back(false);
```

Comment:
@slava77 
the chi2 was only introduced in the 81X wasn't there in the run2-2016 #15916
so it's ok that we have just pick one value -9999 for all the failing